maestro-driver-ios/maestro-driver-iosUITests/Routes/Handlers/ClearTextHandler.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+import FlyingFox
+import os
+import XCTest
+import Network
+
+@MainActor
+struct ClearTextHandler: HTTPHandler {
+    private let typingFrequency = 30
+
+    let logger = Logger(
+        subsystem: Bundle.main.bundleIdentifier!,
+        category: String(describing: Self.self)
+    )
+
+    func handleRequest(_ request: HTTPRequest) async throws -> HTTPResponse {
+        let deleteText = String(repeating: XCUIKeyboardKey.delete.rawValue, count: 40)
+        var eventPath = PointerEventPath.pathForTextInput()
+        eventPath.type(text: deleteText, typingSpeed: typingFrequency)
+        var eventRecord = EventRecord(orientation: .portrait)
+        eventRecord.add(eventPath)
+        try await RunnerDaemonProxy().synthesize(eventRecord: eventRecord)
```

Comment:
Ummm ok thinking out loud, if this call fails I think the runner will crash and we'll have the `XCTestDriverUnreachable` exception right? Do we want to attach some metadata around the exception so that reasons are more clear?

Reply:
For what I saw, the HTTP request will receive a 500. Let me check if the message contains some information.

Reply:
Obviously, the runner should not crash on an invalid request

Reply:
Sounds good. Onboard with idea of contributing in flying fox would be great